Log Viewing/ARKLogTableViewController.m

```diff
@@ -138,6 +141,11 @@ - (void)viewDidLoad;
     self.tableView.rowHeight = 34.0;
     self.tableView.dataSource = self;
     self.tableView.delegate = self;
+    
+    self.searchController = [[UISearchController alloc] initWithSearchResultsController:nil];
```

Comment:
Nope, you have to use the designated initializer, providing nil means you are planning on displaying the results in the same view that you are searching